doc/releases/release-notes-3.4.rst

```diff
@@ -115,6 +115,9 @@ Boards & SoC Support
 Build system and infrastructure
 *******************************
 
+* An issue whereby older versions of the Zephyr SDK toolchain were used
```

Comment:
For consistency with the way release note items are typically worded, please change to:

Fixed an issue whereby [...] instead of the latest

Reply:
Fixed